### PR TITLE
[DOCS-144] Set up progressive disclosure for org roles

### PR DESCRIPTION
**Pull request by mjang-cobalt**
I've been pushing for a Single Source of Truth. But that's not always compatible with UI Text, which needs to be "short and sweet". 

To that end, I'm proposing a hybrid solution, where we include UI Text in the main definition, with progressive disclosure that helps the reader who wants to "Learn more.". Here's how the build looks -- and the reader can select the `>` to disclose the full glossary definition.

<added> @molfinn , Courtney, to try the "build" for yourself, navigate to https://deploy-preview-132--cobalt-docs.netlify.app/getting-started/glossary/#organization-owner

Screen Shot 2022-06-10 at 6 36 47 AM

Note: this is inspired by how k8s does their glossaries, ref https://kubernetes.io/docs/reference/glossary/

### CODE REVIEW

**Comment by mjang-cobalt** [comment]
@ana-dashuk-cobalt I'd like your opinion first, before I share this idea with others from (internal issue) DOCS-144.

**Review by ana-dashuk-cobalt — COMMENTED** [review]
I like this workaround! The definitions are clear for me. I agree that we should keep _Organization_ in the definitions to avoid confusion with _Pentest Team Member_, _Group Owner_, etc.

**Review by ana-dashuk-cobalt — APPROVED** [review]

**Comment by molfinn** [comment]
@mjang Looks good. Outside of this my only feedback was whether "Cobalt Staff" should be italicized to fit the formatting of the other roles. Ex: 

Cobalt Staff
_Cobalt Staff_ members may help you manage the users in your organization. They may also help manage work on your pentests.

Screen Shot 2022-06-10 at 9 33 30 AM

File: content/en/Getting started/glossary.md [hunk 1]
```diff
@@ -149,7 +167,7 @@ A _Pentester_ is a Cobalt pentester who works with a _Pentest Lead_ to test a sp
 
 ### Cobalt Staff
 
-Cobalt Staff members may help you manage the users in your organization. They may also help
+_Cobalt Staff_ members may help you manage the users in your organization. They may also help
```
Comment (mjang-cobalt) [inline content/en/Getting started/glossary.md:170]: Thanks for catching this, @molfinn !

**Pull request merged**